spec.md

```diff
@@ -95,27 +95,19 @@ Both the header and the body of a file are defined in terms of lines.
 A line is a string of text that is terminated with an end-of-line sequence.
 
 ```abnf
+WSP         = <any unicode character with White_Space=yes>
 end-of-line = ( CR / LF / CRLF )
-empty-line  = end-of-line
+empty-line  = *WSP end-of-line
 ```
 
 Implementations SHOULD use a single Line Feed (`%x0A`) as line terminator.
 Implementations MUST accept the end of input (`EOF`) as a valid line terminator.
-Empty lines are ignored throughout the entire file.
-
-> [!CAUTION]
->
-> Whether empty lines are allowed or not is currently open for discussion ([#50](https://github.com/UltraStar-Deluxe/format/issues/50)).
-> Whether a line that consists only of whitespace is recognized as an empty line has not been decided yet.
+Empty lines are ignored throughout the entire file (note that a line consisting only of whitespace characters is considered empty).
```

Comment:
Not sure if we mention this explicitly on the website; if so we'll have to change it there aswell @marwin89

Reply:
I have added a corresponding paragraph to the website. I am honestly not completely sure what part of the spec is _website-worthy_. I think it is the most useful to focus the website mostly on the most common headers and _features_ of the song format. Maybe we could embed the formal spec in the website in a CI step? Or maybe this is best discussed in a separate issue?

Reply:
Preferably the website should at least explain the headers and different note-types and how it should behave. Maybe even by linking towards the spec file. But you're right let's discuss that in a new item, changes for now look good :)